docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -356,6 +356,18 @@ This can cause nodes to become `NotReady` with the following sympotoms:
 
 If more nodes need to be provisioned, either the `clusterNetwork.pods.cidrBlocks` must be expanded or the `node-cidr-mask-size` [should be reduced.]({{< relref "../getting-started/optional/cni/#node-ips-configuration-option" >}}).
 
+### Machine health check shows "Remediation is not allowed"
+
+Sometimes a cluster node is crashed but machine health check does not start the proper remediation process to recreate the failed machine. For example, if a control plane node is crashed, and running `kubectl get mhc ${CLUSTER_NAME}-kcp-unhealthy -n eksa-system -oyaml` shows status message below:
```

Comment:
We should give the example for worker then here right? Or we should mention the defaults for CP below too. 

I think we should make this call out in our regular cluster creation docs as a note, or wherever we talk about cluster-api. But yea maybe there isn't a proper place yet right now.